TrigScint/src/TrigScint/TrigScintTrackProducer.cxx

```diff
@@ -330,8 +330,8 @@ void TrigScintTrackProducer::produce(framework::Event &event) {
 
         // no need to start pulling constituents from tracks that are
         // ridiculously far apart
-        if (fabs(track.getCentroid() - nextTrack.getCentroid() <
-                 3 * maxDelta_)) {
+        if (fabs(track.getCentroid() - nextTrack.getCentroid()) <
+            3 * maxDelta_) {
```

Comment:
@bryngemark I think this is just a misplaced closing parentheses since the newer compiler warned me that `fabs(bool)` doesn't make any sense.

Reply:
i agree with this change